src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -0,0 +1,969 @@
+#pragma warning disable AL0432
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+codeunit 396 NoSeriesManagement
+{
+    ObsoleteReason = 'Please use the "No. Series" and "No. Series - Batch" codeunits instead';
+    ObsoleteState = Pending;
```

Comment:
Is this being removed? If so, where is the CLEAN24 around the whole codeunit?
If it is not being removed, then remove the obsoletion properties after CLEAN24?

Reply:
I guess it's not being completely removed as there are some functions that are not obsoleted

Reply:
Pending. We are still in the process of uptaking this in BaseApp. If we remove it completely before we're done our test builds would fail.